modules/ROOT/pages/upgrading.adoc

```diff
@@ -46,55 +46,105 @@ include::ROOT:partial$_show_page_header_block.adoc[]
 A rolling upgrade is the recommended method to upgrade a Sync Gateway cluster.
 At a high level, a rolling upgrade consists of the following steps:
 
-. The load balancer configuration is updated to stop any HTTP traffic going to the node that will be upgraded.
+. The load balancer configuration stops any HTTP traffic going to the node that's upgraded.
 
-. The upgrade is performed on the given node
+. The upgrade performs on the given node
 
-. The load balancer configuration is updated to re-balance the HTTP traffic across all nodes.
+. The load balancer configuration updates to re-balance the HTTP traffic across all nodes.
 
 Those steps are then repeated for each node in the {sgw} cluster.
 
 
+== Upgrade to 4.0
+
+=== Upgrade Requirements
+
+. Couchbase Server 7.6.0 or later - Sync Gateway 4.0 does not support older server versions.
+. Couchbase Server 7.6.5 or later for bi-directional XDCR features.
```

Comment:
Would it be simpler to only recommend `7.6.5`?

I can't think of a good reason why customers wouldn't be on the latest maintenance version and specifying 7.6.5 as the minimum covers all technical requirements for Sync Gateway.